app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
```

Comment:
- [ ] According to the diagram this should be `belongs_to :author, class_name: 'User'`